dashboard/db/migrate/20170502232017_add_course_id_to_plc_course.rb

```diff
@@ -0,0 +1,5 @@
+class AddCourseIdToPlcCourse < ActiveRecord::Migration[5.0]
+  def change
+    add_column :plc_courses, :course_id, :integer
```

Comment:
Should every `Plc::Course` have a `Course`?  If so let's do `null: false` on the column.